assets/js/atomic/components/product/sale-badge/index.js

```diff
@@ -4,7 +4,7 @@
 import { __ } from '@wordpress/i18n';
 import classnames from 'classnames';
 
-const ProductListSaleBadge = ( { className, product, align } ) => {
+const ProductSalesBadge = ( { className, product, align } ) => {
```

Comment:
Was Sales (plural) an intentional change here? I prefer sale, because it's 'on sale'. Sales also sounds like it should reflect "sales" as in units sold.

Reply:
I agree.  As mentioned in chat, feel free to make that change and merge if your tests confirm things are okay :)

Reply:
Incidentally, maybe change name to `ProductOnSaleBadge`?